airlock/management/commands/run_file_uploader.py

```diff
@@ -11,15 +11,17 @@
 from airlock.business_logic import bll
 from airlock.enums import RequestStatus
 from airlock.types import UrlPath
-from airlock.users import User
 from services.tracing import instrument
+from users.models import User
 
 
 logger = logging.getLogger(__name__)
 
 # We need a user with the output-checker role to access some
 # bll methods
-system_user = User("system", output_checker=True)
+system_user = User(
+    user_id="system", api_data={"username": "system", "output_checker": True}
+)
```

Comment:
Now that this is a db user rather than an ephemeral thing, would it be better to `get_or_create` it within the `handle()` method instead of making it a global (it's only used in one place now anyway)

Reply:
Possibly.

It isn't actually persisted here. We instantiate it directly, and we don't save() it. So it doesn't exist in the db.

I doon't think there's any audit log generated for the bll call we need it for, so the username is not even stored anywhere.

However, having it as a global smells wrong.  I will move it into the handle() call, and add a comment about not actually being persisted.

Reply:
The fix for this is now part of #837